src/plugins/ci.jl

```diff
@@ -60,9 +60,9 @@ source(p::GitHubActions) = p.file
 destination(p::GitHubActions) = joinpath(".github", "workflows", p.destination)
 tags(::GitHubActions) = "<<", ">>"
 
-badges(::GitHubActions) = Badge(
+badges(p::GitHubActions) = Badge(
     "Build Status",
-    "https://github.com/{{{USER}}}/{{{PKG}}}.jl/workflows/CI/badge.svg",
+    "https://github.com/{{{USER}}}/{{{PKG}}}.jl/workflows/$(first(splitext(p.destination)))/badge.svg",
     "https://github.com/{{{USER}}}/{{{PKG}}}.jl/actions",
```

Comment:
i.e. should this change to be like
```suggestion
    "https://github.com/{{{USER}}}/{{{PKG}}}.jl/actions?query=workflow:$(first(splitext(p.destination)))",
```